keymaster/ta/tables.c

```diff
@@ -19,7 +19,48 @@
 
 static keymaster_use_timer_t use_timers[KM_MAX_USE_TIMERS];
 static keymaster_use_counter_t use_counters[KM_MAX_USE_COUNTERS];
-static uint32_t in_use_c;
+static uint32_t in_use_c = 0;
+
+
+/*
+ * This function assumes that keyst from this table are not
+ * to be deleted during TA lifetime
+ */
+keymaster_error_t TA_count_key_uses(uint8_t *key_id,
+				const uint32_t max_uses)
```

Comment:
I have not done this because most of the TA code is written in that style. Personally for me code looks better if everything is aligned, but I was trying to to brake the style of the rest code, and was not sure how to do better here. 
Will fix